doc/source/rllib/package_ref/learner.rst

```diff
@@ -0,0 +1,156 @@
+.. _learner-reference-docs:
+
+Learner API
+===========
+
+Learner specifications and configurations
+-----------------------------------------
+
+.. currentmodule:: ray.rllib.core.learner.learner
+
+.. autosummary::
+    :toctree: doc/
+
+    FrameworkHyperparameters
+    LearnerHyperparameters
+
+TorchLearner configurations
++++++++++++++++++++++++++++
+
+.. autosummary::
+    :toctree: doc/
+
+    TorchCompileWhatToCompile
+
+Constructor
+-----------
+
+.. autosummary::
+    :toctree: doc/
+
+    Learner
+    Learner.build
+    Learner._check_is_built
+    Learner._make_module
+
+Performing Updates
+------------------
+
+.. autosummary::
+    :toctree: doc/
+
+    Learner.update
+    Learner._update
```

Comment:
Learner._update is one of 15 methods of Learner that are annotated with OverrideToImplementCustomLogic.
Some of these are afaics not well represented in the learner doc.
Basically, anything that you should override should also have a description in the doc in the user guide in the Implementation section imo.